posthog/temporal/workflows/snowflake_batch_export.py

```diff
@@ -144,8 +144,17 @@ async def insert_into_snowflake_activity(inputs: SnowflakeInsertInputs):
                     if not result:
                         break
 
+                    # Make sure the properties attribute is parsed as JSON
+                    # TODO: this may well be a change that is worth making in
+                    # the other batch export workflows as well, although I leave
+                    # this as a followup.
```

Comment:
I think this is necessary change, but I don't like that *we* are handling it. The ChClient should return a dict object already. Unless the CH column is typed as String?

Reply:
Can the `ChClient` handle this? Saves us from having to worry about in every single export.

Reply:
Also, this makes it independent of Temporal and the batch export implementation 🤔...

Reply:
Yeah I don't think there is a way in ClickHouse to have the type be JSON. And I don't _think_ there is any way for the client to be instructed to convert to a dict.

I think a better implementation than what we have now is to implement this within the iterator returned from get_results_iterator for the fields that should be unmarshalled, but I was planning on doing that as a follow-up when also handling this in the S3 export, what do you think?